release/images.sh

```diff
@@ -66,23 +81,13 @@ create_image() {
         return 1
     fi
 
-    if [[ -s "${name}.img" ]]; then
-        xz -3 --stdout "${name}.img" > "${final_file}.img.xz"
-        sudo rm "${name}.img"
-
-        if [[ ! -s "${final_file}.img.xz" ]]; then
-            log "Image '${name}'" "Failed to create compressed file."
-            return 1
-        fi
-        sha512sum "${final_file}.img.xz" > "${final_file}.img.xz-${CHKSUM_FILE_SUFFIX}"
-        sign_checksum "${final_file}.img.xz-${CHKSUM_FILE_SUFFIX}"
-    fi
-
-    if [[ -s "${name}.iso" ]]; then
-        mv "${name}.iso" "${final_file}.iso"
-        sha512sum "${final_file}.iso" > "${final_file}.iso-${CHKSUM_FILE_SUFFIX}"
-        sign_checksum "${final_file}.iso-${CHKSUM_FILE_SUFFIX}"
+    if [[ $(echo "${finalize_file}" | wc -w) -gt 1 ]]; then
+        error "Multiple finalize scripts found for ${image}"
+        return 1
+    elif [[ $(echo "${finalize_file}" | wc -w) -eq 1 ]]; then
+        "${finalize_file}"
     fi
+    finalize_images "${name}"
```

Comment:
it looks like you created a function to finalize all the images but is calling it from a function that create a single image.

Reply:
This was named keeping in mind that some images may have multiple format types. I have renamed to finalize_image to make it less confusing.